Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Generic Melee Weapons

### Changelog

- Added (5 rarities x 6 types) weapons
  - 2-handed swords
  - 1-handed swords
  - 2-handed axes
  - 1-handed axes
  - 2-handed maces
  - 1-handed maces
- 1 legendary, 2 epic, 3 rare, 4 uncommon and 5 common weapons of each type will spawn on startup.
- Added naming system for generic weapons: [1st prefix] [X-handed] [2nd prefix] [Weapon], like "Barbaric Two-handed Ritual Sword"

### How to test

- [Proofread this, pls](https://docs.google.com/spreadsheets/d/1FVVfKRCmTzPfiqoKvcfllqOhR613hpCzm2XJ8Gk1moU/edit?usp=sharing "Spreadsheet")

- Check artifacts and their spawn in-game
